docs/guides/qunasys-quri-chemistry.ipynb

```diff
@@ -121,20 +131,228 @@
+    "\n",
+    "The options are explained in the following table.\n",
+    "\n",
+    "| Name                     | Type                        | Description                                                                                                                             | Required | Default |\n",
+    "| :----------------------- | :-------------------------- | :-------------------------------------------------------------------------------------------------------------------------------------- | :------- | :------ |\n",
+    "| particle_number_symmetry | bool                        | If True, remove bitstrings that do not respect particle number conservation from sampling result.                                    | No       | TRUE    |\n",
+    "| spin_symmetry            | bool                        | If True, remove bitstrings that do not respect $s_z$ conservation from sampling result.                                              | No       | TRUE    |\n",
+    "| configuration_recovery   | Union[bool, dict[str, int]] | Activate configuration recovery if set to a value other than False. Refer to the configuration recovery section for dict[str, int] input | No       | FALSE   |\n",
+    "\n",
+    "<Admonition type=\"caution\">You cannot set spin symmetry to True when particle symmetry is False. </Admonition>\n",
+    "\n",
+    "\n",
+    "#### Symmetry post-selection\n",
+    "\n",
+    "These keys control the symmetry post-selection: `particle_number_symmetry` and `spin_symmetry`. Setting `particle_number_symmetry` to True selects those states that conserve the total electron numbers.  Setting `spin_symmetry` to True selects those states that conserve the $s_z$ quantum number. Currently only $s_z = 0$ is supported. The default is:"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
-   "id": "7f910acf-1e6f-4e67-b17d-c3b06de7a562",
+   "execution_count": null,
+   "id": "7dc4540f-92d0-4594-83aa-117cecc51816",
    "metadata": {},
    "outputs": [],
    "source": [
-    "mole = {\n",
+    "mitigation_setting = {\n",
+    "    \"particle_number_symmetry\": True,\n",
+    "    \"spin_symmetry\": True,\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "77e4a0b3-aa4a-48dd-bbe2-40bc6b7cb41f",
+   "metadata": {},
+   "source": [
+    "#### Configuration recovery\n",
+    "\n",
+    "Instead of the symmetry post-selection, you can also choose to do configuration recovery. To turn it on, set it to `True`, and optionally customize it with a dictionary containing the following parameters:\n",
```

Comment:
```suggestion
    "Instead of the symmetry post-selection, you can choose to do configuration recovery. To turn it on, set it to `True`, and optionally customize it with a dictionary containing the following parameters:\n",
```